desktop/core/src/desktop/api_public_urls.py

```diff
@@ -104,6 +104,7 @@
 urlpatterns += [
   re_path(r'^indexer/guess_format/?$', api_public.guess_format, name='indexer_guess_format'),
   re_path(r'^indexer/guess_field_types/?$', api_public.guess_field_types, name='indexer_guess_field_types'),
+  re_path(r'^importer/submit', api_public.importer_submit, name='indexer_importer_submit'),
```

Comment:
Maybe `indexer/importer/submit` to keep the pattern? And same update in docs too?

Reply:
done